go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/nsf/termbox-go v0.0.0-20200418040025-38ba6e5628f1 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/rancher/k3os v0.19.2-dev.4
+	github.com/sirupsen/logrus v1.6.0
```

Comment:
why is the `logrus` needed here?

Reply:
This is missing previously, k3os has logurs vendor so it passed the build without this line. It is added after running go mod tidy, helping to build console package independently